### PR TITLE
add Seth Vargo recap

### PR DESCRIPTION
**Pull request by vanesa**
### Recap blogpost for Seth Vargo's Dev Tool Time episode.

**Preview:** https://deploy-preview-2982--sourcegraph.netlify.app/blog/dev-tool-time-seth-vargo/

**FYI:** The embedded YouTube video is sometimes blocked in the netlify preview but seems to be working when you open the [url](https://www.youtube-nocookie.com/embed/sltjIaAwWac?autoplay=0&amp;cc_load_policy=0&amp;start=93&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=0&amp;rel=0 ) or when you click your way through the whole path [from the start](https://deploy-preview-2982--sourcegraph.netlify.app).
I requested to unblock youtube videos on netlify in the it-tech-ops slack channel but this might not be granted on time. 
Once merged the video _should_ render normally as it renders correctly when I run it locally:
image

### CODE REVIEW

**Comment by content-hacker** [comment]
@vanesa, @rebeccadee is the editor for this post. I added her as a reviewer and removed myself.

**Comment by vanesa** [comment]
@rebeccadee I would love to send out an email with this recap out early in the morning at 9am PT/ 6pm CEST to all the registrants of Seth Vargo's episode. It would be great if we could get this merged before. Please feel free to apply any changes you think are right. I prefer to just fix/change things later than miss out on the opportunity of getting some of the registrants from Seth's episode to sign up for Leah's episode.

This is the copy for the email being sent out: https://app.hubspot.com/email/2762526/edit/45987720476/content

Thank you!

File: blogposts/2021/dev-tool-time-seth-vargo.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+authorUrl: https://twitter.com/vanesacodes
+publishDate: 2021-04-28T10:00-07:00
+tags: [blog]
+slug: dev-tool-time-seth-vargo
+heroImage: https://sourcegraphstatic.com/Twitter-Feed-Seth-Vargo-WS.png
+socialImage: https://sourcegraphstatic.com/Twitter-Feed-Seth-Vargo-WS.png
+published: true
+---
+
+On April 14, 2021, Dev Tool Time welcomed [Seth Vargo](#seth-vargo) as our first guest. You can watch the live stream video on [YouTube Live](https://www.youtube.com/watch?v=sltjIaAwWac) or at the [bottom](#seth-vargo) of this blog post.
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-seth-vargo.md:14]:
```suggestion
Dev Tool Time is our bi-monthly virtual event series in which guest developers share their favorite dev tools and productivity hacks. We welcomed [Seth Vargo](#seth-vargo) as our first guest on April 14, 2021. You can watch the live stream video on [YouTube Live](https://www.youtube.com/watch?v=sltjIaAwWac) or at the [bottom](#seth-vargo) of this blog post.
```

File: blogposts/2021/dev-tool-time-seth-vargo.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+title: Dev Tool Time with Seth Vargo - April 14, 2021
+description: On April 14, 2021, Dev Tool Time welcomed Seth Vargo as our first guest. Seth showed us his productivity optimized desk setup, talked about great window management and keyboard shortcuts. He shared his .gitconfig and much more.
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-seth-vargo.md:3]: Check out the recording of our first episode of Dev Tool Time, in which Google Cloud Engineer Seth Vargo shares his tips for a productivity-optimized desk setup, efficient window management, and keyboard shortcuts.

File: blogposts/2021/dev-tool-time-seth-vargo.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
+title: Dev Tool Time with Seth Vargo - April 14, 2021
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-seth-vargo.md:2]:
```suggestion
title: Dev Tool Time with Seth Vargo: Productivity hacks and .gitconfig tips
```

Comment (rebeccadee) [reply blogposts/2021/dev-tool-time-seth-vargo.md:2]: I think the date of the episode is less important than giving readers an idea of what to expect from the post 😃

File: blogposts/2021/dev-tool-time-seth-vargo.md (hunk 1 above)
Comment (rebeccadee) [reply blogposts/2021/dev-tool-time-seth-vargo.md:14]: Included a brief description of Dev Tool Time for people unfamiliar with it

**Review by rebeccadee — COMMENTED** [review]
@vanesa I'm going to go ahead and apply these suggestions as you mentioned you'd rather get this merged and make edits later. This was in great shape, I just tweaked the title, description and intro a bit to give more context to readers.

File: blogposts/2021/dev-tool-time-seth-vargo.md (hunk 2 above)
Comment (rebeccadee) [reply blogposts/2021/dev-tool-time-seth-vargo.md:3]:
```suggestion
description: Check out the recording of our first episode of Dev Tool Time, in which Google Cloud Engineer Seth Vargo shares his tips for a productivity-optimized desk setup, efficient window management, and keyboard shortcuts.
```

File: blogposts/2021/dev-tool-time-seth-vargo.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+---
+title: Dev Tool Time with Seth Vargo: Productivity hacks and .gitconfig tips
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-seth-vargo.md:2]:
```suggestion
title: "Dev Tool Time with Seth Vargo: Productivity hacks and .gitconfig tips"
```

File: blogposts/2021/dev-tool-time-seth-vargo.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+He also talked about the importance of screen real estate and showed us his desk setup that’s optimized for productivity. Here is a summary of the setup and tools he mentioned:
+
+## Seth’s desk setup for productivity
+
+![Seth Vargo Desk Setup](https://sourcegraphstatic.com/SethVargo-DeskSetup.jpg)
+
+![Seth Vargo Desk Setup](https://sourcegraphstatic.com/SethVargo-DeskSetup2.jpg)
+
+### [LG 5k monitor](https://www.apple.com/shop/product/HMUB2LL/A/lg-ultrafine-5k-display)
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-seth-vargo.md:26]:
```suggestion
### [LG 5K monitor](https://www.apple.com/shop/product/HMUB2LL/A/lg-ultrafine-5k-display)
```

File: blogposts/2021/dev-tool-time-seth-vargo.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+This monitor comes with a built-in USB-C hub to charge his MacBook at 96 watts, his Stream Deck and to connect his [Thunderbolt-based wired Ethernet](https://eshop.macsales.com/item/OWC/TB3ADP10GBE/).
+
+### [Stream Deck](https://www.elgato.com/en/stream-deck)
+
+Each Stream Deck button is assigned to run something, for example one button shows his CPU usage, another the CI status when he’s running a build, while another sends him straight to his GitHub notifications. He loves that it’s highly customizable, adapts to the current app, integrates with a ton of community-built apps, accelerates his workflow, and adds a little fun factor.
+
+![Seth Vargo Stream Deck](https://sourcegraphstatic.com/SethVargo-StreamDeck.jpg)
+
+### [Elgato 4k Cam Link](https://www.elgato.com/en/cam-link-4k)
```
Comment (rebeccadee) [inline blogposts/2021/dev-tool-time-seth-vargo.md:36]:
```suggestion
### [Elgato 4K Cam Link](https://www.elgato.com/en/cam-link-4k)
```

**Review by rebeccadee — APPROVED** [review]

**Comment by rebeccadee** [comment]
> This is the copy for the email being sent out: https://app.hubspot.com/email/2762526/edit/45987720476/content

@vanesa The email copy and settings are all dated April 14 – wanted to check if you linked to the right one?

**Pull request merged**